Lib/_strptime.py

```diff
@@ -565,6 +566,10 @@ def _strptime(data_string, format="%a %b %d %H:%M:%S %Y"):
             hour, minute, second,
             weekday, julian, tz, tzname, gmtoff), fraction, gmtoff_fraction
 
+date_specs = ('%a', '%A', '%b', '%B', '%c', '%d', '%j', '%m', '%U',
```

Comment:
I think this is missing `%G`, `%u` and `%V`, the ISO 8601 week calendar directives.